src/App.js

```diff
@@ -4,15 +4,16 @@ import './App.css';
 import { BrowserRouter as Router, Route, Link } from "react-router-dom";
 import "antd/dist/antd.css";
 import { Alumni } from "./alumni/alumni.component.js"
-
+import CustomForm from './alumni/form.js';
 
 class App extends Component {
   render() {
     return (
       <>
-        <Router>
+        {<Router>
```

Comment:
why did you wrap this with `{`?

Reply:
not sure how that got there. It's been removed.